development/sp-local/config/authsources-pwmanager.php

```diff
@@ -12,7 +12,7 @@
 
     'mfa-idp' => [
         'saml:SP',
-        'entityID' => 'http://pwmanager.local:8084',
+        'entityID' => 'http://pwmanager.local',
```

Comment:
Yeah, given the name of this one (mfa-idp) and the next (mfa-idp-no-port), this one was probably only here for manual local testing/interaction during development, which requires a port number. It might be better to give this some other/different port number for now, but that's certainly not a blocking concern. (My 2 cents.)

Reply:
It was strange that it even had an error. (Why only this one?) I could add it back later and do more diagnostics. Just wanted to keep moving forward.